zanata-war/src/main/java/org/zanata/action/MergeTranslationsManager.java

```diff
@@ -0,0 +1,113 @@
+package org.zanata.action;
+
+import java.io.Serializable;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import org.jboss.seam.ScopeType;
+import org.jboss.seam.annotations.AutoCreate;
+import org.jboss.seam.annotations.In;
+import org.jboss.seam.annotations.Name;
+import org.jboss.seam.annotations.Scope;
+import org.jboss.seam.security.Identity;
+import org.zanata.async.AsyncTaskHandleManager;
+import org.zanata.async.handle.MergeTranslationsTaskHandle;
+import org.zanata.service.MergeTranslationsService;
+
+/**
+ * Manages copy translations from existing version to another tasks.
```

Comment:
Slightly confusing to read, I suggest "Manages tasks to copy translations from one existing version to another."